lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -179,7 +179,8 @@ The function converts the multiline string into a single line string.
 
 sub convert_multiline_string_to_singleline($) {
     my $line = shift;
-    $line =~ s/\n/\\n/sg;
+    # \R will match all Unicode newline sequence
+    $line =~ s/\R/\\n/sg;
```

Comment:
wow this \R character set is cool :-)

Reply:
just discovered it ;) We probably could look for instances of \n and replace them.